ladybug/types/area.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+"""Area data type."""
+from __future__ import division
+
+from ._base import DataTypeBase
+
+
+class Area(DataTypeBase):
+    """Area"""
+    name = 'Area'
+    units = ['m2', 'ft2', 'mm2', 'in2', 'km2', 'mi2', 'cm2', 'ha', 'acre']
+    min = 0
+    abbreviation = 'A'
+
+    def _m2_to_ft2(self, value):
+        return value * 10.7639
+
+    def _m2_to_mm2(self, value):
+        return value * 1000000
+
+    def _m2_to_in2(self, value):
+        return value * 1550
+
+    def _m2_to_km2(self, value):
+        return value / 1000000
+
+    def _m2_to_mi2(self, value):
+        return value / 2590000
+
+    def _m2_to_cm2(self, value):
+        return value * 10000
+
+    def _m2_to_ha(self, value):
+        return value / 10000
+
+    def _m2_to_acre(self, value):
+        return value / 4046.86
+
+    def _ft2_to_m2(self, value):
+        return value / 10.7639
+
+    def _mm2_to_m2(self, value):
+        return value / 1000000
+
+    def _in2_to_m2(self, value):
+        return value / 1550
```

Comment:
@mostaphaRoudsari ,
There's no longer a need to do this now that all of our files have from __future__ import division at their top.  In fact, I would say that the future importing is better since this means that everything is still ok if the input value is an integer and not a float.  Still, I can do this for all integers if you think it makes the code look or feel cleaner.

Reply:
I prefer being explicit in such cases. What if someone/us copies the class around and forgets to import future? You can probably catch those with tests but it is better to take care of it here instead of trying to be careful later.

Reply:
These have all been converted over to floats.